### PR TITLE
Interface name change

### PR DESCRIPTION
**Pull request by arahamad**
@pramodgavali , @depohmel ,  Can you please review these changes

### CODE REVIEW

**Comment by pramodgavali** [comment]
@arahamad I keep seeing so many` linter`  error. As you already there to change  the method name . Can you update the method comments to fix `linter` error . Just interface  methods fine as of now

File: volume-providers/softlayer/block/block_volume_manager_test.go [hunk 1]
```diff
@@ -53,19 +53,19 @@ var _ = Describe("SLBlockSession", func() {
...
-			itemPriceErrors      map[string]error
+			//itemPriceErrors      map[string]error
 			productPackages      []datatypes.Product_Package
 			networkStorages      []datatypes.Network_Storage
 			networkIscsiStorages []datatypes.Network_Storage_Iscsi
 		)
 
 		BeforeEach(func() {
-			capacityItemPrices = []datatypes.Product_Item_Price{
+			/*capacityItemPrices = []datatypes.Product_Item_Price{
```
Comment (depohmel) [inline volume-providers/softlayer/block/block_volume_manager_test.go]: is it not needed anymore?

Comment (arahamad) [reply volume-providers/softlayer/block/block_volume_manager_test.go]: This is the test data. we will check later actually I did not do any changes for this but tried to check `make test` hence got error thats why I commented

**Comment by arahamad** [comment]
as per review comments, I am merging it.

**Pull request merged**